models/intermediate/int_zendesk__comment_metrics.sql

```diff
@@ -8,10 +8,10 @@ comment_counts as (
     select
         ticket_id,
         last_comment_added_at,
-        sum(case when commenter_role = 'internal_comment'
+        sum(case when commenter_role = 'internal_comment' and is_public = true
             then 1
             else 0
-                end) as count_agent_comments,
+                end) as count_public_agent_comments,
```

Comment:
Support for this change can be seen [here](https://support.zendesk.com/hc/en-us/articles/360022182114#topic_sq4_mhq_4y) under "Agent Replies" where multi touch tickets only focus on public agent comments. 

@matthew-skinner do you find non-public agent comments relevant? I ask, because changing the column name will result in a breaking change for the package. If we add the logic but keep the column name then we will be able to retain the same `v0.5.x` version range.

Reply:
My thought was to provide context to what is considered one-touch in Zendesk's documentation/standard by providing that logic in the naming convention, but this could also be added in docs I suppose.

Reply:
Yeah that makes complete sense to me. I am going to keep the naming convention you applied as I agree it is more in line with Zendesk's documentation. However, I am going to retain the old `count_agent_comments` metric (without the `is_public = true`) in addition to including your new `count_public_agent_comments` metric (with `is_public = true`). 

This way we will not need to institute a breaking change, and users will be able to reference total agent comments and total public agent comments, while the total public agent comments will be used in the touch metrics. I can make the changes I just mentioned as well as add some more documentation around the update. I will approve and merge this update to the new base branch. Once I have applied the updates I will open for reviewer and can then merge to master and cut a new release.

Let me know if you have any issue with my planned changes. Thanks!